Sources/JWTKit/Signing/ECDSA/ECDSAKey.swift

```diff
@@ -15,7 +15,7 @@ public final class ECDSAKey: OpenSSLKey {
         where Data: DataProtocol
     {
         let c = try self.load(pem: data) { bio in
-            PEM_read_bio_EC_PUBKEY(convert(bio), nil, nil, nil)
+            CJWTKitBoringSSL_PEM_read_bio_EC_PUBKEY(convert(bio), nil, nil, nil)
```

Comment:
We should be able to remove the `convert` func and any references to it. That may allow us to be a bit more type safe, too!

Reply:
Done!